src/app/modules/gb-export-module/data-table-components/gb-data-table-dimensions/gb-data-table-dimensions.component.ts

```diff
@@ -19,18 +19,16 @@ import {QueryService} from '../../../../services/query.service';
 export class GbDataTableDimensionsComponent implements OnInit {
 
   constructor(private dataTableService: DataTableService,
-              private queryService: QueryService) {
+              private queryService: CohortService) {
```

Comment:
```suggestion
              private cohortService: CohortService) {
```

Reply:
although it seems not to be used, maybe the `CohortService` does not have to be injected here